libs/blocks/merch/merch.js

```diff
@@ -499,19 +499,20 @@ export async function openModal(e, url, offerType, hash, extraOptions, el) {
       window.history.pushState({}, document.title, prevHash !== '' ? `#${prevHash}` : `${window.location.pathname}${window.location.search}`);
     }, { once: true });
   }
-  if (isInternalModal(url)) {
-    const fragmentPath = url.split(/(hlx|aem).(page|live)/).pop();
-    modal = await openFragmentModal(fragmentPath, getModal);
-  } else if (el?.opens3in1Modal) {
+
+  if (el?.opens3in1Modal) {
```

Comment:
nit: I would keep is convention, e.g., `isOpen3in1Modal`

Reply:
Sure, changed 👍